framework/source/class/qx/ui/list/List.js

```diff
@@ -597,6 +625,28 @@ qx.Class.define("qx.ui.list.List",
     },
 
 
+    /**
+     * Event handler for the updated event of the 
+     * qx.ui.virtual.layer.WidgetCell layer.
+     *
+     * Recalculates the item sizes in a deffered call,
+     * which only happens if we have variable item heights
+     */
+    _onLayerUpdated: function () {
+      if (this.isVariableItemHeight() === false) {
+        return;
+      }
+
+      if (this.__deferredLayerUpdate === null) {
+        this.__deferredLayerUpdate = new qx.util.DeferredCall(function () {
+          this._setRowItemSize();
+        }, this);
+      }
+
+      this.__deferredLayerUpdate.schedule();
+    },
+
```

Comment:
@johnspackman Are you sure about that in this case? DeferredCall would only put the identical function reference onto its schedule queue once. This function, however, is anonymous. It could be a non-unique reference on each occurrence, the same (but not identical reference) function to be called multiple times. Unless you're sure, we should verify it.